### PR TITLE
[Fix] Made video controls visible

### LINKED ISSUE #1776: Pop-up does not show video controls on large screens
### Description
On page https://meshery.io/catalog , after clicking **import any catalog item** link there is a pop-up containing video tutorial. So on large screen the video controls are not visible.

### Expected Behavior
The pop-up needs to be responsive on large screens, so it shows video controls.

### Screenshots
On large screens
ss1

On normal screens
ss2

I would like to work on it.

### Enviroment:
 - OS: Mac Linux Windows
 - Browser: Chrome Safari Firefox
 - Version: 
 - Device: Desktop Mobile

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and [designs for Meshery UI](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI) in Figma [(open invite)](https://www.figma.com/team_invite/redeem/qJy1c95qirjgWQODApilR9)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by Muhammed-Moinuddin**
**Description**
Made Pop-up responsive so now video controls are visible.

This PR fixes #1776

**Notes for Reviewers**

**[Signed commits](https://docs.meshery.io/project/contributing#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by vishalvivekm** [comment]
Thank you for your contribution.
Let's discuss this on Websites' call. Adding this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit).

**Comment by Muhammed-Moinuddin** [comment]
> Thank you for your contribution. Let's discuss this on Websites' call. Adding this as an agenda item into the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit).

Sure.

File: _includes/help-modal.html [hunk 1]
```diff
@@ -57,12 +57,20 @@
...
+  margin: 2% auto; /* 5% from the top and centered */
   border: 1px solid #888;
-  width: 60%; /* Could be more or less, depending on screen size */
-  @media (max-width: 768px) {
+  width: 55%; /* Could be more or less, depending on screen size */
+  @media (max-width: 480px) {
     margin: 25% auto;
     width: 85%; /* Change width to 85% */
+  };
+  @media (min-width: 481px) and (max-width: 768px) {
```
Comment (sudhanshutech) [inline _includes/help-modal.html:67]: why use of lot of media queries?

**Review by sudhanshutech — CHANGES_REQUESTED** [review]

File: _includes/help-modal.html (hunk 1 above)
Comment (Muhammed-Moinuddin) [reply _includes/help-modal.html:67]: To make it visible in most of the screens.

Comment (Muhammed-Moinuddin) [reply _includes/help-modal.html:67]: Is this issue valid with you?

Comment (sudhanshutech) [reply _includes/help-modal.html:67]: this much of use of media quries doesn't sound reasonable. Its hard to maintain . These media quries is always hard to manually maintain. There is other css alternative also

Comment (Muhammed-Moinuddin) [reply _includes/help-modal.html:67]: Noted

**Comment by sudhanshutech** [comment]
@Muhammed-Moinuddin add this agenda for website meeting . Lets discuss and finalize this

**Pull request closed without merging**